week-5/Homework/mandatory/3-project/js/main.js

```diff
@@ -0,0 +1,41 @@
+let blueEl = document.getElementById("blueBtn");
+let divEl = document.querySelector(".Jumbotron");
+let donEl = document.querySelector(".btn-primary.btn-lrg");
+let volEl = document.querySelector(".btn-secondary.btn-lrg");
+
+blueEl.addEventListener("click", function () {
+  divEl.style.backgroundColor = "#588fbd";
+  donEl.style.backgroundColor = "#ffa500";
+  volEl.style.backgroundColor = "black";
+  volEl.style.color = "White";
+});
+
+let orangeEl = document.getElementById("orangeBtn");
+// let divEl = document.querySelector(".Jumbotron");
+// let donEl = document.querySelector(".btn-primary.btn-lrg");
+// let volEl = document.querySelector(".btn-secondary.btn-lrg");
+
+orangeEl.addEventListener("click", function () {
+  divEl.style.backgroundColor = "#f0ad4e";
+  donEl.style.backgroundColor = "#5751fd";
+  volEl.style.backgroundColor = "#31b0d5";
+  volEl.style.color = "White";
+});
+
+let greenEl = document.getElementById("greenBtn");
+
+greenEl.addEventListener("click", function () {
+  divEl.style.backgroundColor = "#87ca8a";
+  donEl.style.backgroundColor = "black";
+  volEl.style.backgroundColor = "#8c9c08";
+  volEl.style.color = "White";
+});
+
+let buttonEl = document.querySelector(
+  "#content > div.row > div.primary-content.col-lg-8 > form > button"
```

Comment:
I think you haven't finished second part of the project with form validation. Also you can find a button with a type of submit like this: 
```let buttonEl = document.querySelector("button[type='submit’]");```